CGCharter-1727386911.html

```diff
@@ -41,17 +41,18 @@ <h1>
       </li>
       <li>Previous Charter: <span class="remove">{TBD: URI}</span>
       </li>
-      <li>Start Date: <span class="remove">{TBD: date the charter takes effect,
-      estimate if not known. Update this if the charter is revised and include
-      a link to the previous version of the charter.}</span>
+      <li>Start Date: <span class="remove">1 January 2025</span>
       </li>
-      <li>Last Modifed: 2024-09-26
+      <li>Last Modifed: 2024-12-04
       </li>
     </ul>
     <p>
       The Social Web Incubator Community Group (also known as SocialCG, or
       SWICG) is the successor of the Social Web Working Group, which ran from
```

Comment:
```suggestion
      SWICG) is the successor of the [Social Web Working Group](https://www.w3.org/wiki/Socialwg), which ran from
```

cite for folks who want to click through and learn more history/context.